sdk/core/azure-core/src/http/curl/curl_connection_pool_private.hpp

```diff
@@ -41,6 +44,10 @@ namespace Azure { namespace Core { namespace Http {
     // Give access to private to this tests class
     friend class Azure::Core::Test::CurlConnectionPool_connectionPoolTest_Test;
 #endif
+  private:
+    // The cttor and dttor of this member makes sure of calling the libcurl global init and cleanup
+    AZ_CORE_DLLEXPORT static CurlGlobalStateForAzureSdk CurlGlobalState;
```

Comment:
I don't see any code using it. Should it be moved into an unnamed namespace in a .cpp file?

Reply:
I wanted to be a field of the ConnectionPool, as a way to group them